02_Util/src/util/swagger.ts

```diff
@@ -76,23 +88,19 @@ function OcppTransformObject({
   return openapiObject;
 }
 
-export function initSwagger(
+const registerSwaggerUi = (
   systemConfig: SystemConfig,
   server: FastifyInstance,
-) {
-  server.register(fastifySwagger, {
-    openapi: {
-      info: {
-        title: 'CitrineOS Central System API',
-        description: 'Central System API for OCPP 2.0.1 messaging.',
-        version: '1.1.1',
-      },
-    },
-    transformObject: OcppTransformObject,
-  });
-
+) => {
   const swaggerUiOptions: any = {
     routePrefix: systemConfig.util.swagger?.path,
+    securityDefinitions: {
```

Comment:
need to create authorization swaggerDefinition, but we may have multiple so we may need to move this to OCPI module right @thanaParis ? This is what I was saying is a bit convoluted, because we need the OCPI endpoints to be able to have this authorization securityDefinition